docs/subnet.md

```diff
@@ -145,18 +153,21 @@ With this, we can already create the subnet with `/root` as its parent. We are g
 
 In order to deploy the 3 validators for the subnet, we will have to first export the keys from our root node so we can import them to our validators. Depending on how you are running your rootnet node you'll have to make a call to the docker container, or your nodes API. More information about exporting keys from your node can be found under [this section](#Exporting-wallet-keys).
 
-With the keys conveniently exported, we can deploy the subnet nodes using the `infra-scripts`. The following code snippet showcases the deployment of five Example nodes. Note that each node should be importing a different wallet key for their validator, and should be exposing different ports for their API and validators.
+With the keys conveniently exported, we can deploy the subnet nodes using the `infra-scripts`. Note that each node should be importing a different wallet key for their validator, and should be exposing different ports for their API and validators.
 
-*Example*:
 ```bash
-$ ./bin/ipc-infra/run-subnet-docker.sh 1251 1351 /root/t01002 ~/.ipc-agent/wallet1.key
-$ ./bin/ipc-infra/run-subnet-docker.sh 1252 1352 /root/t01002 ~/.ipc-agent/wallet2.key
-$ ./bin/ipc-infra/run-subnet-docker.sh 1253 1353 /root/t01002 ~/.ipc-agent/wallet3.key
+./bin/ipc-infra/run-subnet-docker.sh <api_port> <validator_port> <subnet_id> <path_to_key>
+```
+```console
+# Example execution
+./bin/ipc-infra/run-subnet-docker.sh 1251 1351 /root/t01002 ~/.ipc-agent/wallet1.key
+./bin/ipc-infra/run-subnet-docker.sh 1252 1352 /root/t01002 ~/.ipc-agent/wallet2.key
+./bin/ipc-infra/run-subnet-docker.sh 1253 1353 /root/t01002 ~/.ipc-agent/wallet3.key
 ```
```

Comment:
Unlike in the other cases these are now missing the prefix; with `console` they will look like output.

Reply:
Fixed